build.jam

```diff
@@ -0,0 +1,35 @@
+# Copyright René Ferdinand Rivera Morell 2023-2024
+# Distributed under the Boost Software License, Version 1.0.
+# (See accompanying file LICENSE_1_0.txt or copy at
+# http://www.boost.org/LICENSE_1_0.txt)
+
+require-b2 5.2 ;
+
+constant boost_dependencies :
+    /boost/assert//boost_assert
+    /boost/concept_check//boost_concept_check
+    /boost/config//boost_config
+    /boost/core//boost_core
+    /boost/detail//boost_detail
+    /boost/function_types//boost_function_types
+    /boost/fusion//boost_fusion
+    /boost/mpl//boost_mpl
+    /boost/optional//boost_optional
+    /boost/smart_ptr//boost_smart_ptr
+    /boost/static_assert//boost_static_assert
+    /boost/type_traits//boost_type_traits
+    /boost/utility//boost_utility ;
+
+project /boost/iterator
+    : common-requirements
+        <include>include
+    ;
+
+explicit
+    [ alias boost_iterator : : : : <library>$(boost_dependencies) ]
+    [ alias all : boost_iterator test ]
```

Comment:
I'm seeing this `test` target in multiple libraries, but I'm not seeing where it is defined. Is this some common Boost-wide target?

Reply:
Ah, I see.